pkg/server/backuprestoreserver.go

```diff
@@ -226,7 +226,7 @@ func (b *BackupRestoreServer) runServer(ctx context.Context, restoreOpts *brtype
 		return nil
 	})
 	if err != nil {
-		return fmt.Errorf("failed to update member peer url: %w", err)
+		b.logger.Errorf("failed to update member peer url: %w", err)
```

Comment:
pipeline checks fails with this `pkg/server/backuprestoreserver.go:229:3: Errorf call has error-wrapping directive %w, which is only supported by Errorf`
I think this is due to this issue: https://github.com/golang/go/issues/47641
```suggestion
		b.logger.Errorf("failed to update member peer url: %v", err)
```

Reply:
Thanks for pointing out
Updated